index.html

```diff
@@ -228,10 +228,10 @@ <h1>
                 <p class="contact-text">
                     If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
                 </p>
-                <form class="my-form">
-                    <input type="text" value="Shakir Hussain">
-                    <input type="email" value="shakir.hussain6969@gmail.com">
-                    <textarea cols="30" rows="10">Write your message here</textarea>
+                <form action="https://formspree.io/f/myyvrwaj" class="my-form" method="post">
+                    <input name="user_name" type="text" placeholder="Name" maxlength="30" required>
+                    <input name="user_email" type="email" placeholder="Email" required>
+                    <textarea name="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
```

Comment:
- [ ] You did a great job working on this section! 🚀 However, one of the main goals of this project is that you demonstrate you're capable of following a given design. Please try to stick as much as possible to the Figma template, this includes colors, fonts, etc.

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/205942457-7dbfbf1c-9ad2-4659-af2e-b6ab199fd5a5.png) | ![image](https://user-images.githubusercontent.com/77069011/205942329-6fc139bd-f938-416d-bc14-51d44ff80eba.png) |


Kindly consider updating the title color and the font of the inputs! 🤓 🛠️